taxcalc/calculate.py

```diff
@@ -91,13 +91,13 @@ def TaxInc_to_AMTI(self):
         AMTI(self.policy, self.records)
 
     def calc_one_year(self):
+        EI_FICA(self.policy, self.records)
```

Comment:
Moved ahead `EI_FICA` function to get `c03260 `, which was now used in calculating `c02900 `.